backend/app/adapter/google/account.go

```diff
@@ -1,46 +1,34 @@
 package google
 
 import (
-	"fmt"
-	"net/http"
 	"short/app/entity"
 	"short/app/usecase/service"
 
+	jwt "github.com/dgrijalva/jwt-go"
```

Comment:
```suggestion
	jwt "github.com/dgrijalva/jwt-go"
```

We shouldn't use `jwt` package here if we are obtaining user profile through `https://openidconnect.googleapis.com/v1/userinfo` API